Lib/idlelib/idle_test/test_sidebar.py

```diff
@@ -612,7 +613,8 @@ def test_interrupt_recall_undo_redo(self):
 
     @run_in_tk_mainloop()
     def test_very_long_wrapped_line(self):
-        with swap_attr(self.shell, 'squeezer', None):
+        with support.adjust_int_max_str_digits(11_111), \
```

Comment:
This test was written by Tal Einat but I am quite sure that there is nothing essential about using an int rather than string literal and that the int simply avoided the need to add quotes.  I would rather replace the current expression, `'x = ' + '1'*10_000 + '\n'` with `f"x = {'a'*10_000!r}\n"` (tested).  However, since you are backporting to 3.7 rather than just 3.10, and have already prepared the backports, I can do this any time after you merge.

Reply:
yep. i'd just let things like this be cleaned up after merge if anyone wants.